build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -102,7 +104,8 @@ module.exports = (yargs) => {
             }
             const {
                 ports,
-                config: { magentoConfiguration, overridenConfiguration: { host, ssl } }
+                config: { magentoConfiguration, overridenConfiguration: { host, ssl } },
+                systemConfiguration: { analytics }
             } = ctx;
 
```

Comment:
Since we now gather analytics data from our users it will be nice to tell them about it, like in many other apps.
I think we should add message about this to the output, so when analytics is enabled we print something like "Yes, we collect analytics data to make our products more stable and reliable! If you want to know more go here (link to docs or smth with collecting data description and with link to the guide how to opt-out)"
If it is disabled, we print according message "Data analytics has been disabled!" or similar.

Reply:
@ejnshtein I guess it make sense to write docs for it now and then insert lines for users about us gathering analytics. Should we insert it each time we are gathering info or just for the first time?

Reply:
I think it should be printed every time so users can enable/disable it whenever they want to! But realistically, after they disable it it's unlikely that they will enable it back, so probably we should print info about gathering data only while it's enabled.